documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -44,8 +44,11 @@ on GitHub.
   when they are supplied as explicit arguments (see <<running-tests-console-launcher-options>>).
 * The new `--details <Details>` option of the `ConsoleLauncher` now allows to select an output
   details mode for when tests are executed. Use one of: `none`, `flat`, `tree` or `verbose`. If
-  `none` is selected, then only the summary and test failures are shown.
-
+  `none` is selected, then only the summary and test failures are shown (see
+  <<running-tests-console-launcher-options>>).
+* An executable `junit-platform-console-{platform-version}-standalone.jar` containing the two
+  test engines Jupiter and Vintage with all dependencies is generated. It provides hassle-free
+  usage of JUnit 5 in projects that manually manage their dependencies.
```

Comment:
Hi @sormuras, that sounds much better, thank you!

I've noticed two little areas for improvement: (1) "Maven central" should be capitalised, (2) "Plain-old" would read better lowercased i.e. "plain-old".

Other than those two areas, I think the new description is ready to be committed. :smile: